WcaOnRails/app/views/panel/index.html.erb

```diff
@@ -34,6 +34,9 @@
       <li>
         <%= link_to "Equipment Funding Form", "https://docs.google.com/forms/d/e/1FAIpQLSeglibvXYG6Z2CzIPbxu2LTpAuXkJPK-vi4GuOnBFx1756ufA/viewform" %>
       </li>
+      <li>
+        <%= link_to "Bands for WCA Dues", "https://www.worldcubeassociation.org/wfc/country-bands" %>
```

Comment:
This will link to prod Dues even on staging. Use the `country_bands_path` helper here instead. It is magically defined by Rails based on the file `config/routes.rb`